examples/with-mocha/test/setupMocha.js

```diff
@@ -0,0 +1,18 @@
+const { cleanup } = require('@testing-library/react')
+const server = require('../mocks/server')
+
+before(() => {
+  server.listen()
+  console.log(`msw server started!`)
+  server.printHandlers()
+})
+
+afterEach(() => {
+  // Reset any modified handlers and cleanup tests
+  server.resetHandlers()
+  cleanup()
```

Comment:
Doesn't RTL call `cleanup` automatically? Does that work only in Jest?

Reply:
It should, but unfortunately, that's not the case here. If you add `console.log(screen.debug())` to the last test in `Child`, you'll see that it stacks components and the tests will fail.